src/Administration/Resources/app/administration/src/module/sw-customer/component/sw-customer-card/index.js

```diff
@@ -137,6 +141,28 @@ export default {
             return this.acl.can('api_proxy_imitate-customer');
         },
 
+        customerImitationWarning() {
+            if (this.customer.guest) {
+                return this.$tc('sw-customer.card.tooltipImitateCustomerGuest');
+            }
+
+            if (!this.customer.active) {
+                return this.$tc('sw-customer.card.tooltipImitateCustomerInactive');
+            }
+
+            if (this.customer.boundSalesChannel) {
+                if (this.customer.boundSalesChannel.typeId !== Defaults.storefrontSalesChannelTypeId) {
```

Comment:
We should also check the `active` state of the sales channel then
```suggestion
            const boundSalesChannel = this.customer.boundSalesChannel;
            if (boundSalesChannel) {
                if (!boundSalesChannel.active || boundSalesChannel.typeId !== Defaults.storefrontSalesChannelTypeId) {
```